.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "recommendations": [
     "dbaeumer.vscode-eslint",
+    "eamodio.tsl-problem-matcher",
```

Comment:
What is this exactly? When I check out this PR, VS Code gives me the following warning:

> The 1 extension(s) below, in workspace recommendations have issues: eamodio.tsl-problem-matcher (not found in marketplace)

Reply:
I would not worry about anything on this branch right now: it's a work in progress that does not run yet.  Speaking directly, that line is unnecessary and will be removed.